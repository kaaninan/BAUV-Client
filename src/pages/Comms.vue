<template>
	<div class="_containerNavigation">
		<div class="row">
			<div class="col-xs-3" data-aos="fade">
				<Connection />
			</div>
			<div class="col-xs-6" data-aos="fade" data-aos-delay='50'>
				<Log />
			</div>
			<div class="col-xs-3" data-aos="fade" data-aos-delay='100'>
			</div>
		</div>
	</div>
</template>

<script>
import AOS from 'aos';
import Connection from '@/components/common/Connection.vue';
import Log from '@/components/common/Log.vue';

export default {
	components: { Connection, Log },
	mounted() {
		AOS.init({once: false})
	}
}
</script>

<style scoped>
	@import '@/assets/styles/variables.css';
	._containerNavigation{
		padding: calc(var(--screen-padding) / 2) var(--screen-padding);
	}
	
</style>