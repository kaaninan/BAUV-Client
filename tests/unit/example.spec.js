import { shallowMount } from '@vue/test-utils'
import Comms from '@/pages/Comms.vue'

describe('Comms.vue', () => {
	// Check Renders correctly
	it('renders correctly', () => {
		const wrapper = shallowMount(Comms)
		expect(wrapper.element).toMatchSnapshot()
	})
})
