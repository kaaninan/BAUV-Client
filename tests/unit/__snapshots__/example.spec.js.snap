// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comms.vue renders correctly 1`] = `
<div
  class="_containerNavigation"
>
  <div
    class="row"
  >
    <div
      class="col-xs-3"
      data-aos="fade"
    >
      <connection-stub
        minimal="false"
      />
    </div>
    <div
      class="col-xs-6"
      data-aos="fade"
      data-aos-delay="50"
    >
      <log-stub />
    </div>
    <div
      class="col-xs-3"
      data-aos="fade"
      data-aos-delay="100"
    />
  </div>
</div>
`;

exports[`MissionControl.vue renders correctly 1`] = `
<div
  class="_containerNavigation"
>
  Mission Control
</div>
`;
